rvo_ros/AgentState state
---
int32 id